src/SessionManager/ProgramSettings.cc

```diff
@@ -76,7 +76,8 @@ ProgramSettings::ProgramSettings(int argc, char** argv) {
  3   warning
  4   info
  5   debug
- 6   trace)");
+ 6   trace
```

Comment:
Should we rename this to `performance` (just here, in the user-facing information)?

It's also apparently possible to create a custom log formatter to change the level name(s) in the logs, but I'm not sure if that's absolutely necessary.

Maybe `trace (performance)`?

Reply:
Agreed. See below for example of how to move from "trace" to "performance"